src/inference/src/dev/core_impl.cpp

```diff
@@ -1274,34 +1274,7 @@ void ov::CoreImpl::set_property_for_device(const ov::AnyMap& configMap, const st
         std::lock_guard<std::mutex> lock(get_mutex());
         created_plugins.reserve(plugins.size());
 
-        // TODO: keep only:
-        //    coreConfig.set_and_update(config);
-        // once GPU remove support of ov::cache_dir
-        // CoreConfg::set_and_update will drop CACHE_DIR from config map
-        // and updates core config with new ov::cache_dir
-        if (deviceName.empty()) {
-            coreConfig.set_and_update(config);
-        } else {
-            OPENVINO_SUPPRESS_DEPRECATED_START
-            auto cache_it = config.find(ov::cache_dir.name());
-            if (cache_it != config.end()) {
-                coreConfig.set_cache_dir_for_device((cache_it->second).as<std::string>(), clearDeviceName);
-                config.erase(cache_it);
-            }
-            OPENVINO_SUPPRESS_DEPRECATED_END
-            // apply and remove core properties
-            auto it = config.find(ov::force_tbb_terminate.name());
-            if (it != config.end()) {
-                auto flag = it->second.as<bool>();
-                ov::threading::executor_manager()->set_property({{it->first, flag}});
-                config.erase(it);
-            }
-
-            it = config.find(ov::enable_mmap.name());
-            if (it != config.end()) {
-                config.erase(it);
-            }
-        }
+        coreConfig.set_and_update(config);
```

Comment:
please, revert this change.
It was supposed to be removed, but currently plugins still need explicitly to know about `cache_dir` property